cosmic_ray/testing/test_runner.py

```diff
@@ -1,4 +1,6 @@
 import abc
+import sys
+import traceback
```

Comment:
I know it's a bit picky, but I'd prefer to keep import groups sorted alphabetically.